Products/CMFPlone/resources/browser/cook.py

```diff
@@ -90,28 +92,30 @@ def cookWhenChangingSettings(context, bundle=None):
                 css_url = siteUrl + '/' + css_resource
                 response = subrequest(css_url)
                 if response.status == 200:
-                    logger.info('Cooking css {0}'.format(css_resource))
+                    logger.info('Cooking css %s', css_resource)
                     css = response.getBody()
-                    cooked_css += '\n/* Resource: {0} */\n{1}\n'.format(
+                    if css_resource[-8:] != '.min.css':
+                        if sys.version_info == (2):
```

Comment:
This conditional returns ``False`` on my python 2 prompt, but the comment on the line below implies that it should be true... maybe you wanted to write ``sys.version_info[0] == 2`` ?